homeassistant/helpers/config_validation.py

```diff
@@ -1040,11 +1041,14 @@ def script_action(value: Any) -> dict:
             ensure_list,
             [
                 {
-                    vol.Optional(CONF_IF): vol.All(ensure_list, [CONDITION_SCHEMA]),
+                    vol.Required(CONF_CONDITIONS): vol.All(
+                        ensure_list, [CONDITION_SCHEMA]
+                    ),
```

Comment:
Now that this is required, would it be good to add another validator, specifically `vol.Length(min=1)`, to make sure users don't forget to add at least one condition (which would effectively make it a default without them possibly realizing it)?

Reply:
I think it's fine to allow empty, that way people might use that to try out their default action.